### PR TITLE
Make it easier for beginners to use hooks

### PR DESCRIPTION
**Pull request by otacke**
Beginners often find it difficult to use hooks, even for seemingly simple things like adding a custom stylesheet. When merged in, the plugin will make it easier to customize H5P by:

- adding and calling a sample stylesheet from the plugin directory
- adding a sample script loaded from the plugin directory (that itself may help to utilize xAPI)
- adding and using utility functions to retrieve fields from semantics
- adding links to documentation

### CODE REVIEW

File: h5pmods.php [hunk 1]
```diff
@@ -157,3 +189,96 @@ function h5pmods_alter_user_result(&$data, $result_id, $content_id, $user_id) {
...
+		// Group
+    array_shift($path_segments);
+    $path_short = implode($path_segments, '/');
+    return find_semantics_path($path_short, $semantics->fields);
+	}
+	else {
+		// Not found
+		return null;
+	}
+}
```
Comment (fnoks) [inline h5pmods.php:244]: Something strange with the indentation here (and below)

Comment (otacke) [reply h5pmods.php:244]: @fnoks Sorry! Copied those two functions from another project that I was working on and it enforced tabs. Fixed.

Comment (fnoks) [reply h5pmods.php:244]: No problemo! Thanks for fixing :)

**Comment by otacke** [comment]
@fnoks Thanks! This merge will make a bunch of people happy tomorrow!

**Pull request merged**